client/src/server/server.js

```diff
@@ -18,7 +18,7 @@ app.get("/estateSchema", (req,res)=>{
     apartments.find().then((apartments)=>{
       res.json(apartments);
       res.status(200);
-      console.log(apartments);
+      console.log('Hello ' + apartments);
     });
```

Comment:
Hey Jörg, I am not sure what this PR is about. You deleted a lot of stuff and changed this log here. Are your main changes on another branch or even merged? I'll have a look on your code on main branch

Reply:
ya, I got git issues. just checkout the main. thanks